controllers/profiles/getProfileByName.js

```diff
@@ -63,13 +63,20 @@ module.exports = async (req, res) => {
     clonedUser.following_symbol = checkMoreOrLess(getFollowingCountResult);
     clonedUser.follower_symbol = checkMoreOrLess(getFollowerCountResult);
 
-    clonedUser.isSignedMessageEnabled = false;
+    if (req?.userId) {
+      clonedUser.is_following = isFollowingResult;
+    }
+
+    clonedUser.isSignedMessageEnabled = true;
     clonedUser.isAnonMessageEnabled = false;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isSignedMessageEnabled` and `isAnonMessageEnabled` flags are set to true and false respectively by default. This is a change from the previous logic where `isSignedMessageEnabled` was set based on whether the user is following or not. If this change is intentional and aligns with your application's requirements, then it's fine. However, if you still want to maintain the previous behavior, you might need to revert this change.



<!-- This is an auto-generated comment by OSS CodeRabbit -->